coldfront/core/resource/views.py

```diff
@@ -197,7 +200,16 @@ def get_queryset(self):
 
         if resource_search_form.is_valid():
             data = resource_search_form.cleaned_data
-            resources = Resource.objects.all().order_by(order_by)
+            if order_by == "name":
```

Comment:
I wonder if we can refactor the logic a bit here? It might be easier to read if we organized it like this:

```
        # These are used elsewhere below so lets initialize them here. We can
        # use the default value of the get() method to set defaults
        order_by = self.request.GET.get('order_by', 'id')
        direction = self.request.GET.get('direction', 'des')
        if order_by != "name":
            if direction == 'des':
                direction = '-'
            order_by = direction + order_by
```

Would something like that work?